src/utils/index.js

```diff
@@ -1,30 +1,73 @@
 import accepts from 'attr-accept'
 
+// Error codes
+export const FILE_INVALID_TYPE = 'file-invalid-type'
+export const FILE_TOO_LARGE = 'file-too-large'
+export const FILE_TOO_SMALL = 'file-too-small'
+export const FILE_EXCESSIVE = 'file-excessive'
+
+// File Errors
+export const REJECTED_INVALID_TYPE = accept => {
```

Comment:
Let's follow the rest of the fn naming convention here (and the rest of the fns):
```suggestion
export const getInvalidTypeRejectionMsg = accept => {
```